src/pages/community/download.md

```diff
@@ -29,14 +29,15 @@ The downloads for the latest release are distributed via mirror sites.
+The latest release of Apache YuniKorn is v0.12.1.
+
+| Version | Release date | Source download                                                                                                                                                                                                                                                                                                                                                                                         | Docker images  | Release notes  |
+|---------|--------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ----  | ----  |
+| v0.12.1 | 2021-12-17   | [Download](https://www.apache.org/dyn/closer.lua/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz) <br />[Checksum](https://downloads.apache.org/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz.sha512) & [Signature](https://downloads.apache.org/incubator/yunikorn/0.12.1/apache-yunikorn-0.12.1-incubating-src.tar.gz.asc)                            | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-0.11.0/images/sha256-7d156e4df2cb1a99d6f3cf5bfd15ae42c7c195f66411b83a720b375194209d20) <br />[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-0.11.0/images/sha256-2b1cee3e79a0f08c835ed264c537b14eb0527d7196dcbbf613296f034c8c2a70 )<br />[web](https://hub.docker.com/layers/apache/yunikorn/web-0.11.0/images/sha256-e07a8465fefb4f51ab989b7be4db824b51fc4b925fb400c09fad87d0b0729246) | [Announcement](/release-announce/0.11.0) |
+| v0.11.0 | 2021-08-18   | [Download](https://www.apache.org/dyn/closer.lua/incubator/yunikorn/0.11.0/apache-yunikorn-0.11.0-incubating-src.tar.gz) <br />[Checksum](https://downloads.apache.org/incubator/yunikorn/0.11.0/apache-yunikorn-0.11.0-incubating-src.tar.gz.sha512) & [Signature](https://downloads.apache.org/incubator/yunikorn/0.11.0/apache-yunikorn-0.11.0-incubating-src.tar.gz.asc)                            | [scheduler](https://hub.docker.com/layers/apache/yunikorn/scheduler-0.11.0/images/sha256-7d156e4df2cb1a99d6f3cf5bfd15ae42c7c195f66411b83a720b375194209d20) <br />[admission-controller](https://hub.docker.com/layers/apache/yunikorn/admission-0.11.0/images/sha256-2b1cee3e79a0f08c835ed264c537b14eb0527d7196dcbbf613296f034c8c2a70 )<br />[web](https://hub.docker.com/layers/apache/yunikorn/web-0.11.0/images/sha256-e07a8465fefb4f51ab989b7be4db824b51fc4b925fb400c09fad87d0b0729246) | [Announcement](/release-announce/0.11.0) |
```

Comment:
The link for the source code for all releases except the latest should point to the archive. That means this link needs to be updated: `https://www.apache.org/dyn/closer.lua/` and should become `https://archive.apache.org/dist/`
Rest stays the same for the 0.11.0 release